doc/deployment/google_cloud_platform.md

```diff
@@ -105,7 +105,7 @@ $ gcloud compute disks list
 $ brew tap pachyderm/tap && brew install pachyderm/tap/pachctl@1.6
 
 # For Linux (64 bit):
-$ curl -o /tmp/pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v1.6.0-RC1/pachctl_1.6.0-RC1_amd64.deb && sudo dpkg -i /tmp/pachctl.deb
+$ curl -o /tmp/pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v1.6.0-RC2/pachctl_1.6.0-RC2_amd64.deb && sudo dpkg -i /tmp/pachctl.deb
```

Comment:
we're on rc4? does this need to be manually updated for every release? I thought we had this managed automatically?